software/fredy.yml

```
@@ -0,0 +1,15 @@
+name: "Fredy"
+website_url: "https://fredy.orange-coding.net/"
+source_code_url: "https://github.com/orangecoding/fredy"
+description: "Fredy - [F]ind [R]eal [E]state [D]amn Eas[y] - Fredy keeps searching for new apartments, houses, and flats in Germany on platforms like ImmoScout24, Immowelt etc and instantly delivers the results to you via Slack, Telegram etc..."
```

Removes application names from description
```suggestion
description: "Searches for new apartments, houses, and flats in Germany on platforms like ImmoScout24, Immowelt, and others, and instantly delivers the results to you via Slack, Telegram, and more."
```